core/src/main/scala/scalacss/Css.scala

```diff
@@ -2,9 +2,12 @@ package scalacss
 
 object Css {
 
-  def apply(styles: TraversableOnce[StyleA])(implicit env: Env): Css =
+  def prepareStyles(styles: TraversableOnce[StyleA])(implicit env: Env): Stream[CssStyleEntry] =
     styles.toStream flatMap styleA
 
+  def prepareKeyframes(styles: TraversableOnce[Keyframes])(implicit env: Env): Stream[CssKeyframesAnimation] =
```

Comment:
Same here. Instead of `Stream[CssKeyframesAnimation]` in many places, could you pls create an alias.